content/en/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -51,6 +51,33 @@ may [fail](https://github.com/kubernetes/kubeadm/issues/31).
 If you have more than one network adapter, and your Kubernetes components are not reachable on the default
 route, we recommend you add IP route(s) so Kubernetes cluster addresses go via the appropriate adapter.
 
+## Ensure iptables tooling does not use the nftables backend
+
+In Linux, nftables is available as a modern replacement for the kernel's iptables subsystem. The
+`iptables` tooling can act as a compatibility layer, behaving like iptables but actually configuring
+nftables. This nftables backend is not compatible with current Kubernetes releases: it causes duplicated
+firewall rules and breaks `kube-proxy`.
+
+If your system's `iptables` tooling uses the nftables backend, you will need to switch the `iptables`
+tooling to 'legacy' mode to avoid these problems. This is the case on at least Debian 10 (Buster),
+Ubuntu 19.04, RHEL and CentOS 8, Fedora 29 and newer releases of these distributions by default.
```

Comment:
I don't believe RHEL 8 supports iptables-legacy at all. See https://access.redhat.com/solutions/4377321 which states that legacy support was dropped. "No, it doesn't. iptables source rpm includes iptables/xtables-legacy-multi.c, however the rpmspec drops legacy ones." if you can't access doc. So we should probably state only for RHEL 7 and, er, say that Kubernetes is not supported on RHEL 8?

Reply:
RHEL 7 doesn't have iptables 1.8, so it only supports legacy mode, so there's nothing to do.

Since RHEL 8 only ships nft mode (which I didn't realize), that means it's not compatible with the official kubernetes kube-proxy container image (which only supports legacy mode), which means it's not compatible with the official kubernetes kubeadm package. But it will work if you build your own packages/images. (It has a new enough iptables 1.8 package that it doesn't hit #82361.)

Reply:
There’s also https://github.com/kubernetes/kubernetes/issues/71305, so it’ll probably mean creating your own kube-proxy image. Is this a scenario we want to document, and if so, should we do that on this page?